dependencies/zlib/meson.build

```diff
@@ -0,0 +1,10 @@
+# ZLib Dependency File
```

Comment:
This is not a dependency upstream anymore (removed by https://github.com/NixOS/nix/commit/15f4d4fd437070e898982c510228b6e8eea17b7d).